src/Annotation.hs

```diff
@@ -94,6 +94,14 @@ data LocTypeAnnot a = LocTypeAnnot { locAnnot :: SRng
 instance HasLoc (LocTypeAnnot a) where
   getLoc = locAnnot
 
+instance HasAnnot LocTypeAnnot where
+  getAnnot = typeAnnot
+  updateAnnot f p = p { locAnnot = locAnnot p, -- does SRNG change here? I presume not...
+                        typeAnnot = f . typeAnnot $ p}
```

Comment:
This could be simplified to just:
```
  updateAnnot f p = p { typeAnnot = f . typeAnnot $ p}
```
since we aren't changing locAnnot.

But also, I'm not sure if we need this instance/if it makes sense. `TypeAnnot` kind of has a different purpose, despite them having (mostly) the same type signature.